cli/viam/main.go

```diff
@@ -45,8 +45,9 @@ func main() {
 	var logger golog.Logger
 
 	app := &cli.App{
-		Name:  "viam",
-		Usage: "interact with your robots",
+		Name:            "viam",
+		Usage:           "interact with your Viam robots through the command line",
```

Comment:
[opt] `through the command line` doesn't seem to add a ton in this context

Reply:
Removed.